ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -218,6 +218,7 @@ jobs:
             {{ common_instance_params(build_test) | indent(12) }}
             GEODE_BRANCH: {{repository.branch}}
             GEODE_FORK: {{repository.fork}}
+            UPSTREAM_FORK: {{repository.fork}}
```

Comment:
`create_instance.sh` should not need UPSTREAM_FORK. Instead, remove the conditional inside of `create_instance.sh` to always an image-family-prefix variable.